server/flags/flags.go

```diff
@@ -18,8 +18,11 @@ const (
 	AppDBBackend = "app-db-backend"
 )
 
+const JSONRPCOnly = "json-rpc-only"
```

Comment:
AFAIK JSON-RPC won't work properly if Tendermint is not enabled. Have you tested this functionality eg by starting a node with this flag enabled and querying a block via JSON-RPC?

Reply:
Not right now, just revert related change.